pycbc/calibration/__init__.py

```diff
@@ -0,0 +1,33 @@
+from pycbc.calibration.recalibrate import Recalibrate
+
+models = {
+    Recalibrate.name : Recalibrate
+}
+
+def read_model_from_config(cp, strain, ifo, section="calibration"):
+    """Returns an instance of the calibration model specified in the
+    given configuration file.
+
+    Parameters
+    ----------
+    cp : WorflowConfigParser
+        An open config file to read.
+    strain : FrequencySeries
+        The strain data that will be recalibrated.
+    ifo : string
+        The detector (H1, L1)  whose model will be loaded.
+    section : {"calibration", string}
+        Section name from which to retrieve the model.
+
+    Returns
+    -------
+    instance
+        An instance of the calibration model class.
+    """
+    name = cp.get_opt_tag(section, "name")
+    model_args = models[name].from_config(cp, ifo)
```

Comment:
Ah wait, here, just have it return the instance of ``Recalibrate`` not the args.

So the lines below here just put those inside ``from_config``.